test/test_util.py

```diff
@@ -48,6 +48,7 @@
 # numbers used for timeouts
```

Comment:
> And I'm sorry for not including the change as a separate commit. I will try to be better.

You can still split it out into its own commit.

> what spurious changes in this file you want to see reverted?

The whitespace changes. They're stylistic and have no bearing on this PR. If you want to submit yet another PR to re-flake8 this project, that's fine, but that should be separate.